File: packages/php/examples/basic.php
<?php

declare(strict_types=1);

require __DIR__ . '/../vendor/autoload.php';

use Talko\Sdk\TalkoClient;

$apiKey = getenv('TALKO_API_KEY');
if ($apiKey === false || $apiKey === '') {
    throw new RuntimeException('Missing TALKO_API_KEY');
}

$talko = new TalkoClient(apiKey: $apiKey);

$conversations = $talko->conversations->list([
    'status' => 'OPEN',
    'limit' => 10,
]);

print_r($conversations);

$connectors = $talko->connectors->list();
print_r($connectors);

File: packages/python/examples/basic.py
from __future__ import annotations

import os

from talko_sdk import TalkoClient


def main() -> None:
    api_key = os.environ.get("TALKO_API_KEY")
    if not api_key:
        raise RuntimeError("Missing TALKO_API_KEY")

    client = TalkoClient(api_key=api_key)

    conversations = client.conversations.list(status="OPEN", limit=10)
    print("Open conversations:", conversations)

    connectors = client.connectors.list()
    print("Configured connectors:", connectors)


if __name__ == "__main__":
    main()
